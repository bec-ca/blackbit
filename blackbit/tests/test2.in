new
protover
sd 10
st 1000000
post
e2e4
quit
