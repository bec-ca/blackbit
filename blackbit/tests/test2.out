feature myname="blackbit" ping=1 usermove=1 draw=0 variants="normal" sigint=0 sigterm=0 setboard=1 playother=1 analyze=1 colors=0 done=1
1 0 0 26 e5
2 24 0 249 e6 Be2
3 -6 1 1303 e6 Be2 Bc5
4 28 2 9399 e5 Nf3 Nf6
5 -2 3 49778 Nc6 Nc3 e5 Bc4 Bb4
6 23 6 161346 Nc6 Nc3 e5 Nf3 Bc5 Bb5
7 -5 20 817445 Nc6 Nc3 Nf6 Bc4 Ne5 Be2 e6
8 24 138 6721617 d5 exd5 Qxd5 Nc3 Qe6 Qe2 Nc6 Nf3
9 0 345 17373359 d5 exd5 Qxd5 Nc3 Qa5 Be2 Nc6 d4 e5
10 18 1763 88302127 Nc6 Nc3 Nf6 d4 d5 e5 Ne4 Nxe4 dxe4 Bb5
move b8c6
